rlberry/agents/torch/ppo/ppo.py

```diff
@@ -207,110 +215,94 @@ def fit(self, budget: int, **kwargs):
             enconters a terminal state in which case it stops early.
```

Comment:
This description of the budget is not true anymore, is it ? Isn't the budget the number of steps ?